### DISCUSSION TITLE
Installation failure from ISO - "install media checksum verification failed"

### DISCUSSION THREAD

**Discussion by sysmedent** [utmstack/UTMStack#807] [Q&A] [answered]
When launching the installer from the utmstack-v10-1-0-autoinstall.iso on a Proxmox VM, the installation fails. It reports "The install media checksum verification failed. It's possible that this crash is related to that checksum failure. Consider verifying the install media and retrying the install."

Confirmed that the md5sum for the ISO matches what's provided in iso/utmstack-v10-1-0-autoinstall.md5 on https://cdn.utmstack.com/:
634e4c320b932a603ac6f6472e524e58  utmstack-v10-1-0-autoinstall.iso

I found this in the installation logs:
ubuntu-server casper-md5check: Checking ./boot/grub/grub.cfg ..... /boot/grub/grub.cfg: mismatch.

In the md5sum.txt on the root directory of the ISO, it has the following entry:
	f617054321761f742970fdd0165e1034  ./boot/grub/grub.cfg

Checking the MD5 sum of the /boot/grub/grub.cfg on the ISO shows it does not match the entry:
    md5sum ./boot/grub/grub.cfg
    fb84b39613316f393030dfda8aa4c329  ./boot/grub/grub.cfg

Is there a problem with the current ISO installer that is available on the website?

**Comment by Celestine-delalt** [comment]
The same problem. Is there already a solution for this?

**Comment by osmontero** [comment]
@c3s4rfred please submit the solution you found over here.

**Comment by KeithSieman** [comment]
Receiving the same issue.

**Comment by c3s4rfred** [comment, marked as answer]
Hi,
Thank you for your report.
We have verified the MD5 checksum of the ISO and do not believe that the mismatch of the GRUB MD5 is the issue, as it is generated with xorriso. This process can sometimes cause checksum differences that don't affect the installation.
For more information, you can refer to the ISO repository here:
https://github.com/utmstack/UTMStack/tree/main/etc/iso.
Please let us know if you encounter further issues or if you need additional assistance.

Best regards
